src/platforms/egs/get_manifests.rs

```diff
@@ -79,12 +57,40 @@ fn is_game_launchable(manifest: &ManifestItem) -> bool {
     (!manifest.launch_executable.is_empty()) || (manifest.is_managed)
 }
 
-fn get_manifest_item(dir_entry: DirEntry) -> Option<ManifestItem> {
+fn get_manifest_item(dir_entry: DirEntry, _path: Option<PathBuf>) -> Option<ManifestItem> {
```

Comment:
Why is this named _path if it is actually used?
```suggestion
fn get_manifest_item(dir_entry: DirEntry, path: Option<PathBuf>) -> Option<ManifestItem> {
```

Reply:
Removing the `_` causes a compiler warning on Windows and I wasn't sure if you'd have checks or workflows preventing builds with compiler warnings.
This just keeps the compiler happy on both platforms.

Reply:
Alright, i can accept that.